deployment.md

```diff
@@ -0,0 +1,225 @@
+# Deployment
+
+There are a number of tools and libraries included in this repository. One is a web server `wm-diffing-server`, which can be deployed to a remote machine to be used in conjuction with the other Web Monitoring projects.
+
+
+## Set up your machine (first time only)
+
+### Set up Python environment
+
+First, SSH into your server, update the package manager, and install `nginx`, `git`, `build-essential`, and `libxml2-dev`. On Debian or Ubuntu Linux, you should run:
+
+```sh
+$ sudo apt-get update
+$ sudo apt-get install git nginx build-essential libxml2-dev
+```
+
+Next, install [`conda`](https://conda.io/), which we’ll use to manage Python versions and environments. You can install either [*Anaconda*](https://www.continuum.io/downloads) (the full-featured version with extra packages and tools) or [*Miniconda*](https://conda.io/miniconda.html) (the minimal, light-weight version). Minconda is recommended to keep the server as simple as possible.
+
+```sh
+# Find the URL of the installer you want to use.
+# For Anaconda, see: https://www.continuum.io/downloads for download URLs
+# For Miniconda, see: https://conda.io/miniconda.html for download URLs
+$ curl <conda_url> > conda_installer.sh 
+# e.g: curl https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh > conda_installer.sh
+```
+
+Then run the installer. Follow the prompts, but instead of the default install location, choose `/opt/conda`. (This gets us a system-wide install so the web server can use it.)
+
+```sh
+$ sudo bash conda_installer.sh
+```
+
+Finally, ensure that you can access conda by creating a new conda group and adding yourself to it.
+
+```sh
+# Create a group for conda and add yourself to it
+$ sudo groupadd conda
+$ sudo usermod -a -G conda <your_username>
+
+# Ensure users in that group have access to conda
+$ sudo chgrp -R conda /opt/conda
+$ sudo chmod 770 -R /opt/conda
+```
+
+
+### Install the application
+
+Next, we need to actually install `web-monitoring-processing` in the `/var/www/web-monitoring-processing` directory:
+
+```sh
+$ cd /var/www
+
+# Make the directory first to ensure it does not get installed for the root user
+$ sudo mkdir web-monitoring-processing
+$ sudo chown <your_user> web-monitoring-processing
+
+# Clone the git repo for the project
+$ git clone https://github.com/edgi-govdata-archiving/web-monitoring-processing.git
+```
```

Comment:
I think this has been wonderfully documented.
Do you think it would be helpful if we put the various commands into a single shell script and run that?
I think that'll make deployment a little easier for new users and seems to be possible to implement here. It's just a suggestion based on my little experience, I'm no expert.